drivers/ethernet/Kconfig.mcux

```diff
@@ -13,64 +13,6 @@ menuconfig ETH_MCUX
 	  configuration change.
 
 if ETH_MCUX
-config ETH_MCUX_PROMISCUOUS_MODE
```

Comment:
These are all driver configurations and should stay here, not move to dts.

Reply:
+1